components/site/src/feeds.rs

```diff
@@ -73,18 +73,23 @@ pub fn render_feed(
     context.insert("config", &site.config.serialize(lang));
     context.insert("lang", lang);
 
-    let feed_filename = &site.config.feed_filename;
-    let feed_url = if let Some(base) = base_path {
-        site.config.make_permalink(&base.join(feed_filename).to_string_lossy().replace('\\', "/"))
-    } else {
-        site.config.make_permalink(feed_filename)
-    };
+    let mut feeds = Vec::new();
+    for feed_filename in &site.config.feed_filenames {
+        let mut context = context.clone();
```

Comment:
That's pretty wasteful.

Can you render them one by one and just updating `context.insert("feed_url", &feed_url);` in the loop? I can't remember what `additional_context_fn(context);` does though

Reply:
`render_template()` takes ownership of the context so I need to clone it on every loop regardless